src/Network/Network/help/Get-AzVirtualHubRouteTable.md

```diff
@@ -8,7 +8,7 @@ schema: 2.0.0
 # Get-AzVirtualHubRouteTable
 
 ## SYNOPSIS
-Gets a Virtual Hub Route Table in a virtual hub or lists all route tables in a virtual hub.
+Gets a Virtual Hub Route Table in a virtual hub or lists all route tables in a virtual hub. The cmdlet [Get-AzVHubRouteTable](./Get-AzRouteTable.md) is replacing this cmdlet.
```

Comment:
Just confirm, you are pointing the link of Get-AzVHubRouteTable to Get-AzRouteTable.md, is that expected

Reply:
Fixed. Thanks for reviewing this.